### PR TITLE
Add parse meta data from http

### PR DESCRIPTION
**Pull request by RooTooZ**
#26 

Now we can use 

```
@JsonApiModelConfig({
    type: 'deals',
    meta: JsonApiMetaModel
})
export class Deal extends JsonApiModel 
```
then
```
export class JsonApiMetaModel {
    public links: Array<any>;

    constructor() {
        this.links = [];
    }
}
```

and if we use `.findAll` we need to use it right that:

```
this.dataStore.findAll(Deal).subscribe((data: JsonApiQueryData<Deal>) => {
            this.deals = data.getModels();
            let meta: JsonApiMetaModel = data.getMeta();
            this.links = meta.links;
}
```

We can define any metadata what we need in **JsonApiMetaModel** like **links** and provide it in annotation to model class

### CODE REVIEW

**Comment by rnikunen** [comment]
looks good to me!

File: src/models/json-api-query-data.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import {JsonApiModel} from './json-api.model';
+
+export class JsonApiQueryData {
```
Comment (HennerM) [inline src/models/json-api-query-data.ts]: can you make this class generic too? thus the need for a typecast when using getModels() would be obsolete.
Besides of this little change the PR LGTM ;)

Comment (RooTooZ) [inline src/models/json-api-query-data.ts]: What did you meant?

Comment (RooTooZ) [inline src/models/json-api-query-data.ts]: Ready!

**Comment by RooTooZ** [comment]
@HennerM can you approve my merge request?

**Comment by HennerM** [comment]
@RooTooZ I can't approve it, only @ghidoz can, he is the owner of this repository.

**Comment by rnikunen** [comment]
Looking for a merge on this... @ghidoz?

**Comment by RooTooZ** [comment]
@HennerM and @rnikunen 

use this feature like this 

```
"dependencies": {
    "angular2-jsonapi": "git+https://git@github.com/RooTooZ/angular2-jsonapi.git",
...
}
```

**Comment by HennerM** [comment]
Thanks for your work.
@ghidoz are you still maintaining this? Otherwise it would be better to have someone other to do it, or maybe just create a fork.

**Comment by clementprdhomme** [comment]
I would also appreciate having an "official" fork of this library so we can merge the PRs of the contributors and fix some of the open issues.

**Comment by kuiperr005** [comment]
Any updates on this issue? When will this be implemented?

**Comment by ghidoz** [comment]
@HennerM unfortunately I was really busy in the last months and I had no time to review the PRs and mantain the package. If you are interested and you have the time, I can make you admin. What do you think?

**Comment by HennerM** [comment]
Thanks @ghidoz for the information.
I would really appreciate if this package gets some maintenance. If you can make me an admin in this repository and an owner of the npm package I am happy to keep this project alive. Especially since there is still a demand for it as seen in #73 and #72 

cc @RooTooZ , @pascal08 @Jcambass

**Comment by pascal08** [comment]
@HennerM that would be great!

**Comment by ghidoz** [comment]
Awesome! @HennerM I added you as a collaborator on this project. What is your username on NPM?
Yes, as I see there are a lot of people who are interested and it would be a pity to waste everything.

**Comment by HennerM** [comment]
Great, thank you very much. My npm username is "hennerm" as well. I will try to go through the issues and PR in the next weeks. If there are other people with more spare-time on their hands I am keen for joining forces, since I am kind of busy in the next days.

**Comment by ghidoz** [comment]
Cool! You are a collaborator also on NPM now. Thanks and let me know if you need to add more collaborators.

**Comment by HennerM** [comment]
Thanks @ghidoz 
but unfortunately it looks like I haven't write access to this repository.

**Comment by ghidoz** [comment]
It says your invitation is pending. I think you should accept it @HennerM

**Comment by HennerM** [comment]
Yeah I see now, thank you!

**Pull request merged**